plugins/infinispan100/src/main/java/org/radargun/service/ServerConfigurationUtils.java

```diff
@@ -0,0 +1,155 @@
+package org.radargun.service;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.stream.Collectors;
+
+import org.apache.commons.io.FileUtils;
+import org.radargun.logging.Log;
+import org.radargun.logging.LogFactory;
+
+import static org.radargun.service.IncludeConfiguration.*;
+
+/**
+ * Infinispan 10.0.x Life cycle
```

Comment:
This is a confusing comment, since RadarGun provides a Liefcycle concept as well. This class seems to be for assembling the server configuration file instead?